src/components/shared/SelectRecipients/SelectRecipientsDialog.tsx

```diff
@@ -66,6 +66,7 @@ export function SelectRecipientsDialogUI(props: SelectRecipientsDialogUIProps) {
         const fuse = new Fuse(items, {
             keys: ['identifier.userId', 'linkedPersona.fingerprint', 'nickname'],
             isCaseSensitive: false,
+            ignoreLocation: true,
```

Comment:
Allowing to search from any location of the word rather than from the first character.